src/components/action/action.tsx

```diff
@@ -44,8 +44,8 @@ export class Action implements InteractiveComponent {
    */
   @Prop({ reflect: true }) alignment?: Alignment;
 
-  /** Specify the appearance style of the action, defaults to solid. */
-  @Prop({ reflect: true }) appearance: Extract<"solid" | "clear", Appearance> = "solid";
+  /** Specify the appearance style of the action. */
+  @Prop({ reflect: true }) appearance: Extract<"solid" | "clear" | "transparent", Appearance>;
```

Comment:
@benelan @jcfranco I think we need a plan on how we can communicate deprecation of values.

Reply:
@anveshmekala do we want to keep "solid" as the default value?

Lets hold off on this one till @jcfranco is back.

Reply:
Deprecation of values should definitely show up in the changelog. We can always make a PR to add additional info to changelog entries (after the corresponding `next` release when the changelog is updated) if there isn't enough space in the PR title. Eventually we want to have changelog-ish page on the developer doc site like the api has release pages.

Reply:
`solid` as default is more meaningful than `default` but it's unnecessary IMO because the expectation of the user is to render the `action` as a whole .  If there is a difference in UX or visual aspect of component when `appearance=solid` vs `appearance=''` then having the `solid`  is understandable.

Reply:
I agree with that, but I think there are a lot of instances where we have the value explicitly set when it is something like "solid" and not assumed. I think we need to maybe set it as "solid" here. cc @jcfranco

Reply:
Let's keep assigning the default value. If the prop wasn't reflected, I'd be OK not assigning to simplify, shed some bits (minimally) and lower selector specificity. Another issue is that the reflected attributes would read as booleans for the unassigned default cases (e.g., `<calcite-button appearance ...>`).

On the plus side, I think we can still keep the simplified CSS (from this PR) by not using the defaults in the selectors. 

> do we want to keep "solid" as the default value?

Great question. I think we should either:

1. always use `default` as the default value
2. change `default` to something more meaningful

Reply:
> Another issue is that the reflected attributes would read as booleans for the unassigned default cases 

Great point. So we need a default value. We just need to change it from "default" to something more meaningful.